src/widget/container.rs

```diff
@@ -0,0 +1,67 @@
+use super::Widget;
+use embedded_graphics::{
+    prelude::*,
+    primitives::{CornerRadii, PrimitiveStyleBuilder, Rectangle, RoundedRectangle},
+};
+
+#[derive(Clone, Copy, Eq, PartialEq)]
+pub struct Border<Color> {
+    pub color: Color,
+    pub width: u32,
+}
+
+#[derive(Clone, Copy, Default, Eq, PartialEq)]
+pub struct Options<Color> {
+    pub background_color: Option<Color>,
+    pub border: Option<Border<Color>>,
+    pub corner_radii: Option<CornerRadii>,
```

Comment:
This seems like overkill (vs a single number), but might as well directly expose the options exposed by embedded_graphics.